service/grails-app/services/org/olf/rs/statemodel/events/EventMessageRequestIndService.groovy

```diff
@@ -245,4 +251,21 @@ public class EventMessageRequestIndService extends AbstractEvent {
             }
         }
     }
+
+    static void mapBibliographicRecordId(Map eventData, Map body, Map newParams) {
+        if (eventData.bibliographicInfo.bibliographicRecordId) {
+            def zflIdentifiers = []
+            def bibliographicRecordId = eventData.bibliographicInfo.bibliographicRecordId
+            if (bibliographicRecordId instanceof ArrayList) {
+                for (def record in bibliographicRecordId) {
```

Comment:
@EddiTim when you figure out how to filter this by the `scheme` attribute, I would like us to add a setting for the value of `scheme`, something like `customIdentifiersScheme=ZFL`